bedrock/firefox/templates/firefox/whatsnew/index.html

```diff
@@ -77,3 +77,9 @@ <h1 class="main-title">{{ _('Download Firefox for your smartphone and tablet.')
 {% block js %}
   {{ js_bundle('firefox_whatsnew') }}
 {% endblock %}
+
+{% block experiments %}
+  {% if switch('whatsnew-78-exp') %}
```

Comment:
We should also add the supported locales to the switch here, to avoid non-experiment locales from being redirected.